db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/river/NotifikasjonHentIdLoeser.kt

```diff
@@ -106,7 +106,7 @@ class NotifikasjonHentIdLoeser(
             )
         } else {
             "Fant ikke sakId.".also {
-                logger.error(it)
+                logger.error(it) // TODO: bare logg en notis?
```

Comment:
Fint med error her så lenge vi har klassen. Det er strengt tatt en feil vi burde være klar over, selv om vi ikke gjør stort for å rette den opp om dagen.